arbnode/sequencer.go

```diff
@@ -406,7 +406,10 @@ func (s *Sequencer) PublishTransaction(parentCtx context.Context, tx *types.Tran
 	}
 }
 
-func (s *Sequencer) preTxFilter(_ *params.ChainConfig, header *types.Header, statedb *state.StateDB, _ *arbosState.ArbosState, tx *types.Transaction, sender common.Address) error {
+func (s *Sequencer) preTxFilter(_ *params.ChainConfig, header *types.Header, statedb *state.StateDB, arbosState *arbosState.ArbosState, tx *types.Transaction, sender common.Address) error {
+	if tx.Type() == types.ArbitrumSubtypedTxType && arbosState.ArbOSVersion() < 11 {
```

Comment:
Need to change this to ArbOSVersion 12

Reply:
I'll address this change in the new version of this PR: https://github.com/OffchainLabs/nitro/pull/1901